src/features/advanced-content-insights/flowTypes.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+export type contentInsightsConfigType = {
+    isActive: boolean,
+    requireEmail: false,
+    requireNotification: false,
+};
+
+export type ContentInsightsSectionTypes = {
+    contentInsightsConfig?: contentInsightsConfigType,
+    /** Whether the advanced content insights FF is enabled */
+    isAdvancedContentInsightsEnabled?: boolean,
+    onAdvancedInsightsEmailToggle?: (isEnabled: boolean) => void,
```

Comment:
I see a mixed usage of `AdvancedContentInsights`, `AdvancedInsights`, `ContentInsights`, could we consolidate these?